notificationworkerlambda/src/main/scala/com/gu/notifications/worker/Harvester.scala

```diff
@@ -60,18 +68,28 @@ trait HarvesterRequestHandler extends Logging {
 
   }
 
+  def routeToSqs: PartialFunction[HarvestedToken, (WorkerSqs, HarvestedToken)] = {
+    case token @ HarvestedToken(_, Ios, _) => (WorkerSqs.IosWorkerSqs, token)
+    case token @ HarvestedToken(_, IosEdition, _) => (WorkerSqs.IosEditionWorkerSqs, token)
+    case token @ HarvestedToken(_, Android, Some(BuildTier.BETA)) => (WorkerSqs.AndroidBetaWorkerSqs, token)
+    case token @ HarvestedToken(_, Android, _) => (WorkerSqs.AndroidWorkerSqs, token)
+    case token @ HarvestedToken(_, AndroidEdition, _) => (WorkerSqs.AndroidEditionWorkerSqs, token)
+  }
+
   def queueShardedNotification(shardedNotifications: Stream[IO, ShardedNotification]): Stream[IO, Unit] = {
     for {
       shardedNotification <- shardedNotifications
-      androidSink = platformSink(shardedNotification, Android, androidLiveDeliveryService)
-      iosSink = platformSink(shardedNotification, Ios, iosLiveDeliveryService)
-      newsstandSink = platformSink(shardedNotification, Newsstand, iosEditionDeliveryService)
-      androidEditionSink = platformSink(shardedNotification, AndroidEdition, androidEditionDeliveryService)
-      iosEditionSink = platformSink(shardedNotification, IosEdition, iosEditionDeliveryService)
+      androidSink = platformSink(shardedNotification, Android, WorkerSqs.AndroidWorkerSqs, androidLiveDeliveryService)
+      androidBetaSink = platformSink(shardedNotification, AndroidBeta, WorkerSqs.AndroidBetaWorkerSqs, androidLiveDeliveryService)
```

Comment:
Should this be `androidBetaDeliveryService`?

Reply:
I'm glad you spotted that one!